### ISSUE TITLE
Libnl detection should be avoided on non Linux systems

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1360]
**Reported by clopez on 20 May 2014 18:46 UTC**

Libnl is a Linux-only library. Therefore it should be avoided to try to use it on non Linux systems.

I got the following error while trying to build aircrack-ng on Debian/kfreebsd:

```
common.mak:85: *** Cannot find development files for any supported version of libnl. install either libnl1 or libnl3..  Stop.
```

**Comment by aircrack-ng** [comment]
**Attachment by clopez on 20 May 2014 18:48 UTC**

Uploaded file [003-dont-abort-libnl-nolinux.diff](https://github.com/aircrack-ng/trac-attachments/blob/master/1360/20140520-184804_171-003-dont-abort-libnl-nolinux.diff)

**Comment by aircrack-ng** [comment]
**Comment by clopez on 20 May 2014 18:46 UTC**

Fixed on aircrack-ng/aircrack-ng@680c0dd57c9ee292ba974d6079b74bd85a84d870
